crates/uv-virtualenv/src/activator/activate_this.py

```diff
@@ -44,13 +45,13 @@
 # prepend bin to PATH (this file is inside the bin directory)
 os.environ["PATH"] = os.pathsep.join([bin_dir, *os.environ.get("PATH", "").split(os.pathsep)])
 os.environ["VIRTUAL_ENV"] = base  # virtual env is right above bin directory
-os.environ["VIRTUAL_ENV_PROMPT"] = "" or os.path.basename(base)  # noqa: SIM222
+os.environ["VIRTUAL_ENV_PROMPT"] = "{{ VIRTUAL_PROMPT }}" or os.path.basename(base)  # noqa: SIM222
 
 # add the virtual environments libraries to the host python import mechanism
 prev_length = len(sys.path)
 for lib in "{{ RELATIVE_SITE_PACKAGES }}".split(os.pathsep):
     path = os.path.realpath(os.path.join(bin_dir, lib))
-    site.addsitedir(path.decode("utf-8") if "" else path)
+    site.addsitedir(path)
 sys.path[:] = sys.path[prev_length:] + sys.path[0:prev_length]
 
 sys.real_prefix = sys.prefix
```

Comment:
I know this is what virtualenv has done for a long time, and that this script is largely vendored from virtualenv, but it's very strange to me that this script monkeypatches a variable onto the `sys` module that wouldn't otherwise exist otherwise. I don't think the stdlib venv module does this, right?

Reply:
I haven't dug into the history deeply, but I think it's a legacy virtualenv thing, possibly from before various evolutions to the `sys` module came about? We don't rely on this anywhere (though `pip` and others have code to detect it). I'm inclined to just leave it as-is to avoid deviations, it doesn't really cost us anything.

Reply:
Yeah. While it's still very weird to me that a script of ours would do this, I don't mind it _too_ much given that virtualenv has been doing it since forever.